README.md

```diff
@@ -51,3 +51,22 @@ environment variable `AFL_NO_CFG_FUZZING` to `1` when building.
 [american-fuzzy-lop]: http://lcamtuf.coredump.cx/afl/
 [AFLplusplus]: https://aflplus.plus/
 [rust]: https://www.rust-lang.org
+
+## IJON
+
+If you want to use IJON - helping fuzzer coverage through code annotation - then
+have a look at the [maze example](afl/examples/maze.rs) how to use it.
+
+You can find the AFL++ IJON documentation at [https://github.com/AFLplusplus/AFLplusplus/blob/stable/docs/IJON.md](https://github.com/AFLplusplus/AFLplusplus/blob/stable/docs/IJON.md)
+
```

Comment:
```suggestion
You can find the AFL++ IJON documentation at [https://github.com/AFLplusplus/AFLplusplus/blob/stable/docs/IJON.md](https://github.com/AFLplusplus/AFLplusplus/blob/stable/docs/IJON.md)

```
Because the documentation will now be linked from IJON a few lines above.